### DISCUSSION TITLE
Seq with ActiveMQ

### DISCUSSION THREAD

**Discussion by AntMassip** [datalust/seq-tickets#1504] [Help]
Hello everyone !

We are trying to put in place a new architecture with microservices and we choose ActiveMQ as a message broker.

We would like to track the messages to see their lifetime 

We could put a logger inside the method which send messages to a queue, is it a good practice to do that ?

So i am wondering if is it a good idea to choose Seq and log4net to log the messages ?  

Thanks ;)

**Comment by liammclennan** [comment]
One way would be to wrap the ActiveMQ client in a type of your own and add the logging in there. You probably don't want to log the entire messages, unless they are very small. Instead log a summary of what the message was.
